decidim-core/app/views/decidim/processes/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<main class="wrapper">
+  <% if hero_processes.any? %>
```

Comment:
It's not styled, so when there are no hero processes I just hide the section. When there are no processes at all it's not styled either, will have to make something up.